### PR TITLE
net-libs/librsync: version bump to 2.0.0, fix #525396

### PR DESCRIPTION
**Pull request by axelgenus**
Refer to bug #525396 (https://bugs.gentoo.org/show_bug.cgi?id=525396).

### CODE REVIEW

**Comment by kensington** [comment]
@gentoo/proxy-maint

**Comment by idella** [comment]
edit 
net-libs/librsync-2.0.0: version bump 
to same as or similar to
net-libs/librsync: bump to version 2.0.0

**Comment by monsieurp** [comment]
LGTM except the missing checksum in the Manifest.

**Comment by idella** [comment]
yes you need add the Manifest file since you are adding a new ebuild, and re-push

File: net-libs/librsync/librsync-2.0.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+HOMEPAGE="http://librsync.sourcefrog.net/"
+SRC_URI="https://github.com/librsync/librsync/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~x86-interix ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos"
+
+RDEPEND="dev-libs/popt"
+DEPEND="dev-libs/popt"
+
```
Comment (gktrk) [inline net-libs/librsync/librsync-2.0.0.ebuild]: wouldn't DEPEND="${RDEPEND}" be easier here?

**Comment by gktrk** [comment]
Just a nitpick: per wiki[1] it's better to use something like:
  Gentoo-Bug: https://bugs.gentoo.org/show_bug.cgi?id=525396
as opposed to:
  Refer to bug #525396 (https://bugs.gentoo.org/show_bug.cgi?id=525396)
And appending the bug number to the first line at the end as such helps with the visibility:
  net-libs/librsync: version bump to 2.0.0 #525396
Like I said, just nitpicking here.

[1] https://wiki.gentoo.org/wiki/Gentoo_git_workflow#Commit_message_format

**Comment by idella** [comment]
runtesting the ebuild yields

>>> Completed installing librsync-2.0.0 into /mnt/gen2/TmpDir/portage/net-libs/librsync-2.0.0/image/

Files matching a file type that is not allowed:
   usr/lib/librsync.so.2.0.0
 * ERROR: net-libs/librsync-2.0.0::gentoo failed:
 *   multilib-strict check failed!

suggesting it was run with FEATURES=-multilib-strict.
I suggest you add multilib-strict to the FEATURES var in your make.conf.
You will need get the fix / solution from the mutlilb eclass

**Pull request merged**